app/views/users/show.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Here you can see all the info of the user</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<section id="user_data">
+  <%= render partial: "shared/userbar", locals: {
+    photo: @user.photo,
+    username: @user.name,
+    post_number: @user.posts_counter
+  } 
+  %>
+  <div class="bio-container">
+    <p class="title">Bio</p>
+    <p class="text"> <%= @user.bio %></p>
+  </div>
+
+  <%=render partial: "shared/postbar", locals: {posts: @user.posts} %>
+</section>
+
```

Comment:
- [ ] **See all Post** links open user's post index as displayed on the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=ANIPACIRNXBW3VN5L6SQGRLBSU4WG)